### DISCUSSION TITLE
Error: auth/invalid-api-key: Firebase: Error (auth/invalid-api-key).

### DISCUSSION THREAD

**Discussion by thiwa8** [AntlerVC/firetable#1553] [Support / Q&A]
For a month I have been trying to access the buildship database (see orange arrow below).
I invariably receive that error

![Screenshot buildship 2024-05-07 210253](https://github.com/rowyio/rowy/assets/47010732/b615920b-2e46-47c3-aacf-521e3156fedf)

**Comment by Chadha93** [comment]
Hi @thiwa8, could you please submit your project via in-app support form? https://docs.buildship.com/support-messages. We'll fix the db access issue asap for your project.
